test/examples/examples.test.ts

```diff
@@ -0,0 +1,109 @@
+import { createNextDescribe } from 'e2e-utils'
+import path from 'path'
+import fs from 'fs-extra'
+
+const testedExamples = [
```

Comment:
Instead of using an allowlist, we might want to change this to an ignorelist so that new examples are automatically tested.

Reply:
Since we are not running in normal CI it's actually possible now 👍

We probably should run all examples and we will see in the CI exactly which ones are broken.

But I still think that we want to have a workflow like this with only 1st party examples that should be always green.

And if we start running CI on all examples we need to be extra cautious when adding new libraries because we will be installing that library with access to our CI secrets.

Reply:
Will add this in Follow-up PR